lib/rules/multiline-ternary.js

```diff
@@ -18,10 +18,15 @@ module.exports = {
             category: "Stylistic Issues",
             recommended: false
         },
-        schema: []
+        schema: [
+            {
+                enum: ["always", "never"]
+            },
```

Comment:
@kaicataldo is correct, this is unnecessary. There's no need for additionalProperties:false.